vault/resource_aws_secret_backend.go

```diff
@@ -356,26 +323,8 @@ func awsSecretBackendUpdate(ctx context.Context, d *schema.ResourceData, meta in
 	if err != nil {
 		return diag.FromErr(err)
 	}
-	if d.HasChanges(consts.FieldDefaultLeaseTTL, consts.FieldMaxLeaseTTL, consts.FieldDescription, consts.FieldIdentityTokenKey) {
-		description := d.Get(consts.FieldDescription).(string)
-		config := api.MountConfigInput{
-			Description:     &description,
-			DefaultLeaseTTL: fmt.Sprintf("%ds", d.Get(consts.FieldDefaultLeaseTTL)),
-			MaxLeaseTTL:     fmt.Sprintf("%ds", d.Get(consts.FieldMaxLeaseTTL)),
-		}
-
-		if useAPIVer116 {
-			identityTokenKey := d.Get(consts.FieldIdentityTokenKey).(string)
-			if identityTokenKey != "" {
-				config.IdentityTokenKey = identityTokenKey
-			}
-		}
-		log.Printf("[DEBUG] Updating mount config input for %q", path)
-		err := client.Sys().TuneMountWithContext(ctx, path, config)
-		if err != nil {
-			return diag.Errorf("error updating mount config input for %q: %s", path, err)
-		}
-		log.Printf("[DEBUG] Updated mount config input for %q", path)
+	if err := updateMount(d, meta, true); err != nil {
```

Comment:
Should we also pass a context here to updateMount()?

Reply:
Updated all update instances to also use the context